registry/random/functions.go

```diff
@@ -50,9 +50,9 @@ func (rr *RandomRegistry) RandAlpha(count int) string {
 //
 //	string - the randomly generated ASCII string.
 //
-// Example:
+// For an example of this function in a go template, refer to [Sprout Documentation: randAscii].
 //
-//	{{ 10 | randAscii }} // Output: "}]~>_<:^%" (output will vary)
+// [Sprout Documentation: randAscii]: https://docs.atom.codes/sprout/registries/random#randascii
```

Comment:
Here and everywhere: Why using Markdown note, while you could use directly a link?

```diff
-// For an example of this function in a go template, refer to [Sprout Documentation: randAscii]. 
-// [Sprout Documentation: randAscii]: https://docs.atom.codes/sprout/registries/random#randascii
+ // For an example of this function in a go template, refer to [Sprout Documentation: randAscii](https://docs.atom.codes/sprout/registries/random#randascii).
```

Is it a sugar syntax or the rendering differs?

Reply:
The behaviors and display differ after rendering due to the following reasons:

- "Markdown direct" `[]()` → Interpreted as a link during HTML rendering.
- "Markdown reference" `[]:` → Interpreted as a link during Markdown rendering (this format is enforced by `gofmt`).

Source : [Go Doc](https://tip.golang.org/doc/comment#:~:text=Links%C2%B6,in%20HTML%20renderings.)

Reply:
Thanks for the explanation I'll look further for some personnel project.